src/main/java/org/opentestsystem/delivery/logging/ConfigRefreshListener.java

```diff
@@ -0,0 +1,38 @@
+package org.opentestsystem.delivery.logging;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.core.Message;
+import org.springframework.amqp.core.MessageListener;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Registered in spring to listen to amqp messages on the "springCloudBus" topic.
+ * An anonymous queue is created and bound to the topic.
+ *
+ * When spring cloud config server detects a change in configuration, it sends a message
+ * to the "springCloudBus" topic.
+ *
+ * When the message is received, the logback configuration is reloaded
+ * from the centralized spring cloud config server.
+ */
+public class ConfigRefreshListener extends LoggingConfigRefresher implements MessageListener {
+  private static final Logger logger = LoggerFactory.getLogger(ConfigRefreshListener.class);
+  private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I feel this should be injected via Spring (constructor not field).  I guess if it is null then you could create a new ObjectMapper but all the applications that will use this refresher already have an ObjectMapper configured.